src/components/DestinationButton.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import {
+  StyleSheet,
+  Text,
+  TextStyle,
+  TouchableOpacity,
+  ViewStyle,
+  Dimensions,
+  View,
+} from 'react-native';
+import { Ionicons } from '@expo/vector-icons';
+
+const WIDTH = Dimensions.get('window').width;
+
+export default function DestinationButton(props) {
```

Comment:
Maybe place this component file in src/screens/map/ since it's specific to the map screen?

Reply:
@griffisr